proj/src/main/webapp/script.js

```diff
@@ -3,69 +3,48 @@
  */
 
 // Elements of login container
-const fname = document.getElementById("fname")
 const txtEmail = document.getElementById("email");
 const txtPassword = document.getElementById("pass");
 const btnLogin = document.getElementById("btnLogin");
 const btnSignUp = document.getElementById("btnSignUp");
 const btnLogout = document.getElementById("btnLogout");
 
 // Add login event
-if(btnLogin){
-    btnLogin.addEventListener("click", e => {
-        const emailVal = txtEmail.value;
-        const passVal = pass.value;
+btnLogin.addEventListener("click", e => {
+    const emailVal = txtEmail.value;
+    const passVal = pass.value;
 
-        // Initialize auth object
-        const auth = firebase.auth();
+    // Initialize auth object
+    const auth = firebase.auth();
 
-        const promise = auth.signInWithEmailAndPassword(emailVal, passVal).then(function(user){
-            window.location.replace("chat.html");
-        });
-        promise.catch(e => console.log(e.message));
-    });
-}
+    const promise = auth.signInWithEmailAndPassword(emailVal, passVal);
+    promise.catch(e => console.log(e.message));
+});
 
-// Add sign up event
-if(btnSignUp){
-    btnSignUp.addEventListener("click", e => {
-        const emailVal = txtEmail.value;
-        const passVal = pass.value;
-
-        // Initialize auth object
-        const auth = firebase.auth();
-        auth.useDeviceLanguage();
-
-        const promise = auth.createUserWithEmailAndPassword(emailVal, passVal).then(function(){
-            auth.currentUser.updateProfile({displayName: fname.value + " " + lname.value}).then(function(){
-                console.log("display name updated successfully");
-                window.location.replace("chat.html");
-            }).catch(function(){
-                console.log("error updating display name");
-            })
-        });
-        promise.catch(e => console.log(e.message));
+// Add sign up event 
+btnSignUp.addEventListener("click", e => {
+    const emailVal = txtEmail.value;
+    const passVal = pass.value;
```

Comment:
I'd expect there to be a confirm password field when signing up that you make sure is the same as the password field.